### DISCUSSION TITLE
How to get access_token !?

### DISCUSSION THREAD

**Discussion by realtebo** [nextauthjs/next-auth#12230] [Help]
I am using Hubspot as provider, but I think it's a common use case.

After user is loggedin, I need to make call using access_token of the user.

Where can I get it!?

**Comment by Giuliao** [comment]
Hi, I have also met the same problem. below are the  answers get from  the ai agent,
<img width="781" alt="image" src="https://github.com/user-attachments/assets/4244dbfb-664d-4b1f-bc04-8b41db13a338">

> For specific configuration, refer to  https://stackoverflow.com/questions/69068495/how-to-get-the-provider-access-token-in-next-auth

The way ai agent showed dose help, but the access token only exits when the user first login, 
<img width="746" alt="image" src="https://github.com/user-attachments/assets/77899f54-08da-4903-9257-85426c9a5fcb">

So I am looking for help too. How i can get the access token, so that i can access the other resource by token from the provider like github.

@realtebo @linyiru

**Reply by Giuliao** [reply]
I solved the problem..In my code, I should use token.accessToken not token.access_token in session function in auth.ts/js. Sorry for bothering.
